core/templates/dev/head/pages/exploration_player/tutor_card_directive.html

```diff
@@ -140,6 +140,7 @@
   .conversation-skin-tutor-card .instructions-button {
     background: inherit;
     border: none;
+    white-space:normal;
```

Comment:
nit: add space after colon

Reply:
Hi @tjiang11 , I will correct it . Thanks for reviewing .